packages/grid/_modules_/grid/components/columnHeaders/GridColumnHeaderItem.tsx

```diff
@@ -100,6 +101,13 @@ export const GridColumnHeaderItem = ({
     [publish],
   );
 
+  const resizeEventHandlers = React.useMemo(
+    () => ({
+      onMouseDown: publish(GRID_COLUMN_RESIZE_MOUSE_DOWN),
```

Comment:
can we move that to `mouseEventsHandlers` and just call it on `GRID_COLUMN_HEADER_LEAVE_MOUSE_DOWN` as it's just a mouse down triggered by the column item, that you just use to achieve the resize functionality.

Reply:
These events are assigned to a different element. The `mouseEventsHandlers`  are added to the root and `resizeEventHandlers` are passed to the column header separator. it looks cleaner if they are separate.

Reply:
I didn't see with the code folded 🤔

Reply:
how about `GRID_COLUMN_SEPARATOR_MOUSE_DOWN` seems more in our convention.

Reply:
No real preference for following React (mouse down) vs. the DOM (mousedown). A few days later, I would be slightly leaning towards +1 for following the React naming convention because it's what most React developers are exposed to. They use the DOM API less frequently.

Reply:
Updated